changelogs/fragments/8648-fix-gitlab-runner-paused.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "gitlab_runner - fix ``paused`` parameter being ignored (https://github.com/ansible-collections/community.general/pull/8648)"
```

Comment:
```suggestion
  - "gitlab_runner - fix ``paused`` parameter being ignored (https://github.com/ansible-collections/community.general/pull/8648)".
```

Reply:
thanks, i added the missing dot inside the string double quotes (after checking the other fragments)

Reply:
Oh sorry, indeed, it should have been inside the quotation :)